packages/application-shell/src/components/custom-view-shell/custom-view-shell.tsx

```diff
@@ -0,0 +1,142 @@
+import {
+  useCallback,
+  useEffect,
+  useRef,
+  useState,
+  type ReactNode,
+} from 'react';
+import { PageUnauthorized } from '@commercetools-frontend/application-components';
+import {
+  type ApplicationWindow,
+  CUSTOM_VIEWS_EVENTS_NAMES,
+} from '@commercetools-frontend/constants';
+import {
+  AsyncLocaleData,
+  type TAsyncLocaleDataProps,
+} from '@commercetools-frontend/i18n';
+import type { TCustomView } from '../../types/generated/settings';
+import ApplicationLoader from '../application-loader/application-loader';
+import ApplicationShellProvider from '../application-shell-provider';
+import { getBrowserLocale } from '../application-shell-provider/utils';
+import ConfigureIntlProvider from '../configure-intl-provider';
+import CustomViewAuthenticatedShell from '../custom-view-authenticated-shell';
```

Comment:
Nit: for naming consistency, it should be `custom-view-shell-authenticated`

Reply:
Renamed here: 9446cae41fa966890bae8d2f49c790bf0fda335d